<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioDialog</class>
 <widget class="QDialog" name="ScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenario Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="icon_la">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>81</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../../.designer/icons/cplus_logo.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_123">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Showing the saved scenario information and analysis output details if available.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Scenario Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_64">
     <property name="text">
      <string>Scenario name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="scenario_name">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_71">
     <property name="toolTip">
      <string>Information about the scenario analysis, limited to 195 characters. </string>
     </property>
     <property name="text">
      <string>Scenario description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="scenario_description">
     <property name="toolTip">
      <string>Information about the scenario analysis, limited to 195 characters. </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>195</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="group_box_aoi">
     <property name="title">
      <string>Area of Interest</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_studyarea">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_extent">
        <property name="text">
         <string>Extent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" alignment="Qt::AlignTop">
       <widget class="QStackedWidget" name="studyarea_stacked_widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QLineEdit" name="scenario_studyarea">
         <property name="toolTip">
          <string>Path to the study area layer. </string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QgsExtentGroupBox" name="extent_box">
         <property name="title">
          <string>Area of interest (Study Area)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="collapsed">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCrsdescription">
     <property name="text">
      <string>Scenario CRS for analysis (Must be projected CRS)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="crs_text"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
